contributing.md

```diff
@@ -459,13 +459,13 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
 
 1. In your `build` folder, compile with `-DCMAKE_BUILD_TYPE=Coverage`
 
-        cmake -DCMAKE_BUILD_TYPE=Coverage ..\
+        cmake -DCMAKE_BUILD_TYPE=Coverage ../
```

Comment:
It looks like the `colcon` vs `cmake` difference made another victim, sorry about that :confused: 

See another related PR: https://github.com/ignitionrobotics/docs/pull/40

Your change on line 467 suggests you're in a colcon workspace. Where are you running this command in that case?

Reply:
Yes, I was a bit confused at first. I'm using a colcon workspace. I also tested the `cmake` command in a build folder in the source directory of the package. Should I make this more clear in the doc?

Reply:
I don't believe that the slash should be necessary in either case, correct?

Reply:
No it's not necessary

Reply:
> Should I make this more clear in the doc?

Yeah I think we can err on the side of being more verbose here, but cover all use cases.

So we should explain that, if the user is in a colcon workspace, they should do

```
cd <path to colcon workspace>
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Coverage
```

And if they're using plain cmakle

```
cd <path to build directory>
cmake -DCMAKE_BUILD_TYPE=Coverage ..
```

Reply:
Made clarification changes here 14dd9e53b57ead6f9584d971a495dfb4bcd16c2a